Earthfile

```diff
@@ -317,12 +317,33 @@ image-rootfs:
 
 
 ## UKI Stuff Start
+enki-image:
+    FROM  quay.io/kairos/enki:v0.0.8
+    SAVE ARTIFACT /enki enki
+
+uki-iso:
+        ARG --required BASE_IMAGE # BASE_IMAGE is existing kairos image which needs to be converted to uki
+        FROM $BASE_IMAGE
+        ARG ISO_NAME=$(cat /etc/os-release | grep 'KAIROS_ARTIFACT' | sed 's/KAIROS_ARTIFACT=\"//' | sed 's/\"//')
+
+        FROM +uki-dev-tools-image
```

Comment:
The enki image ` quay.io/kairos/enki:v0.0.8` has all the dependencies already. You can use that as `FROM`, not just to copy the enki binary from it.

Reply:
@jimmykarily That container has an entrypoint so it's not really picking up the RUN command properly. Do you know how to do this? I tried to use the `RUN --entrypoint` but no lock